src/main/java/com/tobmistaketracker/TobMistake.java

```diff
@@ -24,7 +24,9 @@ public enum TobMistake {
     VERZIK_P2_ACID("Verzik P2 Acid", "verzik_p2_acid.png", "I can't count to four!"),
     VERZIK_P3_MELEE("Verzik P3 Melee", "verzik_p3_melee.png", "I'm PKing my team!"),
```

Comment:
Lol I just noticed I actually already had the mistake for this modeled. We should use this text for if the melee actually goes off. I can work on trying to add that though separately. No need to worry about it.